AirBench/Controllers/BenchWebApiController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using AirBench.Data;
+using AirBench.FormModels;
+using AirBench.Models;
+using AirBench.Repository;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Web;
+using System.Web.Http;
+
+namespace AirBench.Controllers
+{
+    [RoutePrefix("api/Bench")]
+    public class BenchWebApiController : ApiController
+    {
+        private IBenchApiRepository Repository;
+        Context context = new Context();
+
+        public BenchWebApiController()
+        {
+            Repository = new BenchApiRepository(context);
+        }
+
+        public BenchWebApiController(IBenchApiRepository repo)
+        {
+            Repository = repo;
+        }
+        [Route("Get")]
```

Comment:
You shouldn't add `Get` to your endpoint path/route. If you're going to use attribute routing, change your method definition to this:

```
[Route("")]
[HttpGet]
async public Task<BenchResponse> Get()
```